roadmap/src/components/Roadmap/Roadmap.js

```diff
@@ -1,88 +1,149 @@
-              <div
-                className="roadmap-cards-item-content"
-                dangerouslySetInnerHTML={{ __html: t.description }}
-              />
-            </div>
-          ))}
-        </div>
-      </div>
-      <div className="roadmap-column">
-        <div className="roadmap-column-heading">Done</div>
-        <div className="roadmap-cards">
-          {doneTickets.map((t, i) => (
-            <div key={`exploring-${i}`} className="roadmap-cards-item">
-              <div className="roadmap-cards-item-heading">{t.title}</div>
-              <div
-                className="roadmap-cards-item-content"
-                dangerouslySetInnerHTML={{ __html: t.description }}
-              />
-            </div>
-          ))}
-        </div>
-      </div>
-      <div className="roadmap-column">
-        <div className="roadmap-column-heading">Leaving it for now</div>
-        <div className="roadmap-cards">
-          {leavingItForNowTickets.map((t, i) => (
-            <div key={`exploring-${i}`} className="roadmap-cards-item">
-              <div className="roadmap-cards-item-heading">{t.title}</div>
-              <div
-                className="roadmap-cards-item-content"
-                dangerouslySetInnerHTML={{ __html: t.description }}
-              />
+      {loading ? (
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          style={{
+            margin: "auto",
+            background: "none",
+            display: "block",
+            shapeRendering: "auto",
+            maxWidth: "30px",
+            marginTop: "-20px",
+          }}
+          width="200px"
+          height="200px"
+          viewBox="0 0 100 100"
+          preserveAspectRatio="xMidYMid"
+        >
+          <circle
+            cx="50"
+            cy="50"
+            fill="none"
+            stroke="#4d5273"
+            strokeWidth="10"
+            r="35"
+            strokeDasharray="164.93361431346415 56.97787143782138"
+          >
+            <animateTransform
+              attributeName="transform"
+              type="rotate"
+              repeatCount="indefinite"
+              dur="1s"
+              values="0 50 50;360 50 50"
+              keyTimes="0;1"
+            ></animateTransform>
+          </circle>
+        </svg>
+      ) : (
+        <>
+          {columnMap.map((column, i) => (
+            <div className="roadmap-column" key={`${column.title}-${i}`}>
+              <div className="roadmap-column-heading">{column.title}</div>
+              <div className="roadmap-cards">
+                {column.tickets.map((t, i) => (
+                  <div key={`exploring-${i}`} className="roadmap-cards-item">
+                    <div className="roadmap-cards-item-heading">{t.title}</div>
+                    <div
+                      className="roadmap-cards-item-content"
+                      dangerouslySetInnerHTML={{ __html: t.description }}
+                    />
+                    <svg
```

Comment:
Import the svg file 
Ex: `import { ReactComponent as UpvoteIcon } from '../icons/upvote.svg'`